aperture_properties.py

```diff
@@ -2755,6 +2933,22 @@ class ApertureProperties(HaloProperty):
             "BHmaxvel",
             "BHmaxAR",
             "BHmaxlasteventa",
+            "BlackHolesTotalInjectedEnergy",
```

Comment:
One small thing: could you add an equivalent of this quantity, `BlackHolesTotalInjectedJetEnergy`, that is defined as the sum over all injected jet energies from all the BHs?

Reply:
https://github.com/SWIFTSIM/SOAP/pull/97/commits/f77ce2d6be819765a685be2618366cac76ba19e5